### PR TITLE
Code smells resolved

### PR DESCRIPTION
**Pull request by swetatech24**

### CODE REVIEW

File: src/main/java/com/iemr/mmu/service/dataSyncActivity/DownloadDataFromServerImpl.java [hunk 1]
```diff
@@ -192,32 +159,25 @@ private int downloadDataFromServer(SyncDownloadMaster syncDownloadMaster, String
...
-				// JSONArray dataList = obj.getJSONArray("data");
-				// Consume the response from API in master data digester
 				MasterDownloadDataDigester masterDownloadDataDigester = InputMapper.gson().fromJson(response.getBody(),
 						MasterDownloadDataDigester.class);
 				if (masterDownloadDataDigester != null) {
 					int i = updateMastersWithLatestData(masterDownloadDataDigester, syncDownloadMaster);
 					if (i > 0)
 						successFlag = 1;
 				} else {
-					// error in parsing response data
```
Comment (harimohanr) [inline src/main/java/com/iemr/mmu/service/dataSyncActivity/DownloadDataFromServerImpl.java:204]: @Anjana2310 @swetatech24 In this class you have removed some valid comments too.

File: src/main/java/com/iemr/mmu/service/common/transaction/CommonNurseServiceImpl.java [hunk 2]
```diff
@@ -519,7 +491,6 @@ private ArrayList<FemaleObstetricHistory> getFemaleObstetricHistoryObj(
 		StringBuilder postAbortionComp;
 		StringBuilder postAbortionCompValues;
 
-		// iterate through pregnancy complication
 		for (FemaleObstetricHistory obj : femaleObstetricHistorylist) {
```
Comment (Anjana2310) [inline src/main/java/com/iemr/mmu/service/common/transaction/CommonNurseServiceImpl.java:494]: Add the API description back

File: src/main/java/com/iemr/mmu/service/common/transaction/CommonNurseServiceImpl.java [hunk 3]
```diff
@@ -2084,8 +2009,6 @@ public int updateBenPastHistoryDetails(BenMedHistory benMedHistory) throws Parse
 		Integer r = 0;
 		int delRes = 0;
 		if (null != benMedHistory) {
-			// Delete Existing past History of beneficiary before inserting
-			// updated history
 			ArrayList<Object[]> benMedHistoryStatuses = benMedHistoryRepo
```
Comment (Anjana2310) [inline src/main/java/com/iemr/mmu/service/common/transaction/CommonNurseServiceImpl.java:2012]: Add  the previous desciption

File: src/main/java/com/iemr/mmu/service/common/transaction/CommonNurseServiceImpl.java [hunk 4]
```diff
@@ -2606,14 +2529,13 @@ public Long savePrescriptionDetailsAndGetPrescriptionID(Long benRegID, Long benV
...
-		if(provisionalDiagnosisList != null)
+		if (provisionalDiagnosisList != null)
 			prescriptionDetail.setProvisionalDiagnosisList(provisionalDiagnosisList);
 
 		Long prescriptionID = saveBenPrescription(prescriptionDetail);
 		return prescriptionID;
 	}
 
-	// save prescription of covid19
 	public Long savePrescriptionDetailsCovid19(Long benRegID, Long benVisitID, Integer psmID, String createdBy,
```
Comment (Anjana2310) [inline src/main/java/com/iemr/mmu/service/common/transaction/CommonNurseServiceImpl.java:2539]: Add // save prescription of covid19

File: src/main/java/com/iemr/mmu/service/common/transaction/CommonNurseServiceImpl.java [hunk 5]
```diff
@@ -2689,7 +2603,6 @@ public Long saveBenPrescription(PrescriptionDetail prescription) {
 	public int updatePrescription(PrescriptionDetail prescription) {
 		int i = 0;
 
-		// SnomedCT new code
 		StringBuilder pdTerm = new StringBuilder();
```
Comment (Anjana2310) [inline src/main/java/com/iemr/mmu/service/common/transaction/CommonNurseServiceImpl.java:2606]: Add // SnomedCT new code

File: src/main/java/com/iemr/mmu/service/common/transaction/CommonNurseServiceImpl.java [hunk 6]
```diff
@@ -3001,13 +2913,10 @@ public int saveBenInvestigationDetails(WrapperBenInvestigationANC wrapperBenInve
 			wrapperBenInvestigationANC.setPrescriptionID(prescriptionID);
 			investigationSuccessFlag = saveBenInvestigation(wrapperBenInvestigationANC);
 			if (investigationSuccessFlag != null && investigationSuccessFlag > 0) {
-				// Investigation data saved successfully.
 				res = 1;
```
Comment (Anjana2310) [inline src/main/java/com/iemr/mmu/service/common/transaction/CommonNurseServiceImpl.java:2916]: Add // Investigation data saved successfully.

File: src/main/java/com/iemr/mmu/service/common/transaction/CommonNurseServiceImpl.java [hunk 7]
```diff
@@ -3001,13 +2913,10 @@ public int saveBenInvestigationDetails(WrapperBenInvestigationANC wrapperBenInve
...
 			} else {
```
Comment (Anjana2310) [inline src/main/java/com/iemr/mmu/service/common/transaction/CommonNurseServiceImpl.java:2917]: Add // Something went wrong !!!

File: src/main/java/com/iemr/mmu/service/common/transaction/CommonNurseServiceImpl.java [hunk 8]
```diff
@@ -3001,13 +2913,10 @@ public int saveBenInvestigationDetails(WrapperBenInvestigationANC wrapperBenInve
...
-				// Something went wrong !!!
 			}
 		} else {
-			// Invalid Data..
 		}
```
Comment (Anjana2310) [inline src/main/java/com/iemr/mmu/service/common/transaction/CommonNurseServiceImpl.java:2920]: Add // Invalid Data..

File: src/main/java/com/iemr/mmu/service/common/transaction/CommonNurseServiceImpl.java [hunk 9]
```diff
@@ -261,39 +260,37 @@ public Long saveBeneficiaryVisitDetails(BeneficiaryVisitDetail beneficiaryVisitD
...
+			}
 		}
 		return i;
 	}
 
 	public Long generateVisitCode(Long visitID, Integer vanID, Integer sessionID) {
 		String visitCode = "";
 
-		// van & session ID
 		String vanIDString = "";
```
Comment (Anjana2310) [inline src/main/java/com/iemr/mmu/service/common/transaction/CommonNurseServiceImpl.java:294]: Add // van & session ID

**Review by Anjana2310 — COMMENTED** [review]
Add back the API descriptions in CommonNurseServiceImpl

File: src/main/java/com/iemr/mmu/service/common/transaction/CommonNurseServiceImpl.java [hunk 10]
```diff
@@ -381,18 +369,14 @@ public BeneficiaryVisitDetail getCSVisitDetails(Long benRegID, Long visitCode) t
...
 		ArrayList<Map<String, String>> fileList = new ArrayList<>();
 		if (benVisitDetailsOBJ != null && benVisitDetailsOBJ.getReportFilePath() != null
 				&& benVisitDetailsOBJ.getReportFilePath().trim().length() > 0) {
 
 			String fileIdsTemp[] = benVisitDetailsOBJ.getReportFilePath().split(",");
-			// fileIds = new String[fileIdsTemp.length];
 			for (String str : fileIdsTemp) {
 				if (str != null && str.trim().length() > 0) {
-					// DE40034072,20-04-2022, decrypting internal file path
 					String decryptedFilePath = null;
```
Comment (Anjana2310) [inline src/main/java/com/iemr/mmu/service/common/transaction/CommonNurseServiceImpl.java:380]: Description missing.

File: src/main/java/com/iemr/mmu/service/common/transaction/CommonNurseServiceImpl.java [hunk 11]
```diff
@@ -1805,8 +1733,7 @@ public BenMenstrualDetails getMenstrualHistory(Long beneficiaryRegID, Long visit
 		ArrayList<Object[]> menstrualHistory = benMenstrualDetailsRepo.getBenMenstrualDetail(beneficiaryRegID,
 				visitCode);
 		BenMenstrualDetails menstrualHistoryDetails = BenMenstrualDetails.getBenMenstrualDetails(menstrualHistory);
-
-		// CRs changes, 30-10-2018
+		// CRs changes
```
Comment (Anjana2310) [inline src/main/java/com/iemr/mmu/service/common/transaction/CommonNurseServiceImpl.java:1736]: You can remove this comment as it is not meaningful.

File: src/main/java/com/iemr/mmu/service/common/transaction/CommonNurseServiceImpl.java [hunk 12]
```diff
@@ -2606,14 +2532,13 @@ public Long savePrescriptionDetailsAndGetPrescriptionID(Long benRegID, Long benV
...
-		if(provisionalDiagnosisList != null)
+		if (provisionalDiagnosisList != null)
 			prescriptionDetail.setProvisionalDiagnosisList(provisionalDiagnosisList);
 
 		Long prescriptionID = saveBenPrescription(prescriptionDetail);
 		return prescriptionID;
 	}
 
-	// save prescription of covid19
 	public Long savePrescriptionDetailsCovid19(Long benRegID, Long benVisitID, Integer psmID, String createdBy,
```
Comment (Anjana2310) [inline src/main/java/com/iemr/mmu/service/common/transaction/CommonNurseServiceImpl.java:2542]: Add back the API description comment

File: src/main/java/com/iemr/mmu/service/common/transaction/CommonNurseServiceImpl.java [hunk 13]
```diff
@@ -3752,7 +3627,6 @@ public int updateBenPhysicalActivityHistoryNCDScreening(PhysicalActivityType phy
 	public String getBenSymptomaticData(Long benRegID) throws Exception {
 		Map<String, Object> responseMap = new HashMap<>();
 		ArrayList<Map<String, Object>> ansList = new ArrayList<>();
-		// get date before 3 month
 		java.util.Date referenceDate = new java.util.Date();
```
Comment (Anjana2310) [inline src/main/java/com/iemr/mmu/service/common/transaction/CommonNurseServiceImpl.java:3630]: Add the comment back

File: src/main/java/com/iemr/mmu/service/common/transaction/CommonNurseServiceImpl.java (hunk 10 above)
Comment (Anjana2310) [inline src/main/java/com/iemr/mmu/service/common/transaction/CommonNurseServiceImpl.java:380]: comment missing

File: src/main/java/com/iemr/mmu/service/labtechnician/LabTechnicianServiceImpl.java [hunk 14]
```diff
@@ -480,7 +425,6 @@ public Integer saveLabTestResult(WrapperLabResultEntry wrapperLabResults) {
 				labResultEntry.setVanID(wrapperLabResults.getVanID());
 				labResultEntry.setParkingPlaceID(wrapperLabResults.getParkingPlaceID());
```
Comment (Anjana2310) [inline src/main/java/com/iemr/mmu/service/labtechnician/LabTechnicianServiceImpl.java:427]: comment missing

File: src/main/java/com/iemr/mmu/service/pnc/PNCServiceImpl.java [hunk 15]
```diff
@@ -153,16 +147,9 @@ public Long savePNCNurseData(JsonObject requestOBJ) throws Exception {
 			Long examtnSaveSuccessFlag = null;
 			Integer i = null;
 
-			// temporary object for ben flow part. for getting visit reason and
-			// category and ben reg id
 			JsonObject tmpOBJ = requestOBJ.getAsJsonObject("visitDetails").getAsJsonObject("visitDetails");
```
Comment (Anjana2310) [inline src/main/java/com/iemr/mmu/service/pnc/PNCServiceImpl.java:150]: comment missing

File: src/main/java/com/iemr/mmu/service/pnc/PNCServiceImpl.java [hunk 16]
```diff
@@ -580,17 +552,14 @@ public Long saveBenPNCHistoryDetails(JsonObject pncHistoryOBJ, Long benVisitID,
...
 			} else {
 				// Female Obstetric Details not provided.
 			}
 
 		} else {
 			obstetricSuccessFlag = new Long(1);
 		}
-		
-		/** For Female above 12 and below 16 years.. **/
 		// Save Immunization History
```
Comment (Anjana2310) [inline src/main/java/com/iemr/mmu/service/pnc/PNCServiceImpl.java:563]: Add back the comment

File: src/main/java/com/iemr/mmu/service/pnc/PNCServiceImpl.java [hunk 17]
```diff
@@ -869,7 +828,6 @@ public Long saveBenExaminationDetails(JsonObject examinationDetailsOBJ, Long ben
 		return exmnSuccessFlag;
 	}
 
-	// ----------Fetch ANC (Nurse) --------------------------------------
```
Comment (Anjana2310) [inline src/main/java/com/iemr/mmu/service/pnc/PNCServiceImpl.java:831]: comment missing

**Review by Anjana2310 — COMMENTED** [review]
@swetatech24 Please address the review comments

File: src/main/java/com/iemr/mmu/service/common/transaction/CommonNurseServiceImpl.java (hunk 10 above)
Comment (swetatech24) [reply src/main/java/com/iemr/mmu/service/common/transaction/CommonNurseServiceImpl.java:380]: It is like this in PSMRI

**Pull request closed without merging**